tests/gemini_test.yaml

```diff
@@ -18,6 +18,9 @@ gemini_cmd: "gemini -d --duration 10800s -c 20 -p 100 -m mixed -f"
 gemini_version: 'latest'
```

Comment:
@aleksbykov please give the yaml more descriptive name.
something like gemini_3h_nemesis
BTW, it shouldn't replace the current one, it should be additional